fake_useragent/fake.py

```diff
@@ -128,12 +142,22 @@ def __getattr__(self, attr):
 
             attr = attr.lower()
 
-            if attr == 'random':
-                browser = random.choice(self.data_randomize)
+            if attr in ['random', 'mobile', 'desktop']:
+                if attr == 'mobile':
+                    browser = random.choice(list(self.data_browsers_mobile.keys()))
+                elif attr == 'desktop':
+                    browser = random.choice(list(self.data_browsers_desktop.keys()))
+                else:
+                    browser = random.choice(self.data_randomize)
```

Comment:
what about skipping one level of `if` nesting by using `elif` here - like you've done below?

```python
if attr == 'random':
  browser = random.choice(self.data_randomize)
elif attr == 'mobile':
  browser = random.choice(list(self.data_browsers_mobile.keys()))
elif attr == 'desktop':
  browser = random.choice(list(self.data_browsers_desktop.keys()))
else:
  browser = random.choice(self.data_randomize)
```